spacy/pipeline/entity_linker.py

```diff
@@ -490,6 +498,8 @@ def load_model(p):
         deserialize["kb"] = lambda p: self.kb.from_disk(p)
         deserialize["model"] = load_model
         util.from_disk(path, deserialize, exclude)
+        if "overwrite" not in self.cfg:
+            self.cfg["overwrite"] = BACKWARD_OVERWRITE
```

Comment:
Does this one still make sense?

Reply:
No. It won't do anything, but I meant to remove it.